app/models/skin.rb

```diff
@@ -188,6 +188,7 @@ def cache!
     self.public = true
     self.official = true
     save!
+    Rails.cache.increment('skins_generation/'+(self.type.nil? ? ("site_skin") : self.id.to_s) )
```

Comment:
Are helpers significantly slower ( To my c mind are they function calls or a macro )